plugins/postcss-cascade-layers/src/index.ts

```diff
@@ -4,13 +4,7 @@ const creator: PluginCreator<{ color: string }> = (opts?: { color: string }) =>
 	return {
 		postcssPlugin: 'postcss-cascade-layers',
 		Declaration(decl) {
-			if (decl.value === 'red') {
-				if (opts && opts.color) {
-					decl.value = opts.color;
-				} else {
-					decl.value = 'blue';
-				}
-			}
+			decl.remove();
```

Comment:
The demo transforms actually made the test pass as it checks for `green`.
Removing declarations made it fail as wanted.